lighthouse-core/lib/manifest-parser.js

```diff
@@ -298,27 +298,26 @@ function parseApplication(raw) {
 
 function parseRelatedApplications(jsonInput) {
   const raw = jsonInput.related_applications;
-  let value;
 
   if (raw === undefined) {
     return {
       raw,
-      value,
+      value: undefined,
```

Comment:
it didn't like that I was using `value` as `undefined` by default and then reassigning it to a value below. It considered only the explicit assignment a thing, and only counted it as a single assignment, so thought it should be `const`...but that meant I had to manually set it to `undefined` above or just assign these in place